R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -76,10 +76,10 @@ if (override_eval) {
 We use default settings in which it runs in local mode. It auto downloads Spark package in the background if no previous installation is found. For more details about setup, see [Spark Session](#SetupSparkSession).
 
 ```{r, include=FALSE}
-install.spark()
+#install.spark()
```

Comment:
Out of curiosity why is this needed?

Reply:
`pkgdown` builds vignettes, not only `Rd` files, so `install.spark` fails, because we don't have corresponding release yet.

Reply:
It is just a temporary workaround ‒ I'll fix it, once I get it to build outside local environment (I am thinking about adding environment variable that can be set in CI).